Makefile

```diff
@@ -96,7 +95,10 @@ gen-openapi: $(OPENAPI_GEN)
 gen-crds: $(OPERATOR_SDK)
 	$(OPERATOR_SDK) generate crds
 
-handler: gen-openapi gen-k8s gen-crds $(OPERATOR_SDK)
+manifests:
+	$(GO) run hack/render-manifests.go nmstate $(HANDLER_IMAGE) Always deploy/ $(manifests)
```

Comment:
Please set all the attributes via a changeable env variable. I don't have a good usecase for it now, but it may save us some PR iterations in the future if we need to generate manifests for different namespace for instance.

Reply:
Ok, seting nmstate as env var, pullPolicy not so, it's ok ?